engine/database.py

```diff
@@ -16,21 +16,21 @@ def __init__(self, robot):
             heading_pid: heading PID in format of [proportional factor, integral factor, derivative factor]           
```

Comment:
I think this is just for simulating noise (I don't think it led to any fruitful results, just "our simulation doesn't work well if there is a lot of noise") - if that's the case, not sure if it should be a dedicated attribute

Reply:
does this mean we don't want to have a heading pid then?

Reply:
wait what. I think we need a heading pid. heading pid should be different from traversal pid

Reply:
i asked julie on slack and i think she meant testing for noise in general. Since database is going to be scrapped since this refactoring allows us to replace/rewrite it we can prob account for it later

Reply:
so are we still talking about heading pid or heading noise?